locale/en/blog/weekly-updates/weekly-update.2015-11-13.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Weekly Update - Nov 13th, 2015
+author: Minwoo Jung (@jmwsoft)
+date: 2015-11-13T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2015-11-13
+layout: blog-post.hbs
+---
+
+### Node.js News — November 13th
+
+### Node.js Foundation Individual Membership Now Open
+
+* With the support of the Linux Foundation we are now able to launch an Individual Membership program. 
+* These members will be electing two representatives to the Board of Directors this January who will be responsible for representing the diverse needs of the Node.js community in the administration of the Node.js Foundation.
+
+See https://nodejs.org/en/blog/community/individual-membership/ for more information.
+
+### SF Node Meetup
+
+* ["The Journey To Node 4" by Joe McCann](https://www.youtube.com/watch?v=jZpnF0zOSvY): Node.js, io.js, the Node Foundation, Node v4, etc.
+* ["Our Journey With Node.js" by Mick Ryan](https://www.youtube.com/watch?v=uFY_p1YZ2cE): How Node.js was introduced at BigCommerce, pitfalls, benefits, etc.
+* ["YOU Are Invited"](https://www.youtube.com/watch?v=1F8F-EbdX40): Rich Trott encouraged people to contribute to Node and tried to point out some non-obvious on-ramps.
+
+### Community Updates
+
+* [10 Habits of a Happy Node Hacker (2016)](http://blog.heroku.com/archives/2015/11/10/node-habits-2016), "Here are ten habits for happy Node.js hackers as we enter 2016. They're specifically for app developers"
+* [Understanding Garbage Collection and hunting Memory Leaks in Node.js](http://apmblog.dynatrace.com/2015/11/04/understanding-garbage-collection-and-hunting-memory-leaks-in-node-js/), "Covers how Node.js manages memory and how to trace down memory-related problems"
+* [Node.js v4 gets an A+ for SSL Labs with no configuration](https://certsimple.com/blog/node-js-ssl-labs), "Node has one of the best out-of-the-box SSL setups of any web server."
+
+If you have spotted or written something about Node.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues/191), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [Nodevember](http://nodevember.org/?utm_source=io.js+and+Node.js+News&utm_medium=article), November 14th - 15th at Nashville, Tennessee, US.
```

Comment:
If we're sending this out in the next 12 hours or so, it might be OK. People who see it won't be able to go, but it's good for people to know that these events are happening all the time. (And maybe someone can check the Nodevember site for a live stream.)